ui/yarn.lock

```diff
@@ -1480,7 +1480,7 @@
     sass "^1.26.11"
     tippy.js "^6.3.1"
 
-"@hashicorp/structure-icons@^1.9.0-0", "@hashicorp/structure-icons@^1.9.2":
+"@hashicorp/structure-icons@^1.9.2":
```

Comment:
**[observation, non-blocking]**

As with ember-svg-jar, pds-ember depends on this and so keeps it around. Not the end of the world.